### ISSUE TITLE
Can't reproduce the result on French

### ISSUE THREAD

**Issue by OfirArviv** [SUDA-LA/ucca-parser#3]
Hi,

First of all, thank you for adding the BERT code!
I tried to reproduce the multilingual-bert experiment and I can't manage to reproduce the result on French. 
I run 5 instances of the parser and the average result on the Fr -test was 0.7506 . Abut 0.2 points below what you got.
I used the default configurations, and the training stopped early in epochs 22 to 32. 
Did you changed any of the configuration when you reproduces your results? Maye didn't use early stopping?

Would appreciate any help!

Thanks,
Ofir

**Comment by HMJW** [comment]
First, please make sure your code is same with the multilingual-lexical-bert branch in the repository and the versions of ucca and pytorch module are same with the requirements. Because the newest ucca and pytorch module have changed a lot, I guess it may have some effects (I saw the requirements.txt in your repository).

I didn't change any of the default configuration when I reproduced the results in the README.md (I ran 2 instances of the parser today again and the results are both 77.0 till this moment). I am sorry that I don't realize the results would be so unstable because I do few experiments on French. Except the running environment, I think there may be other several reasons as follows:

1. Too little French training data.
2. The previous code in the paper is slightly different from the current version. The previous code uses the average of subwords vectors output by BERT, but the current version just chooses the first subword vector as feature for simplicity.
3. The model evaluates the hybrid development set during training. It may be better to evaluate just on French development set.

I recommend you to reinstall the specified versions of the packages (python==3.6.8, ucca==1.0.127, pytorch==1.0.1, and pytorch-pretrained-bert==0.6.2) and re-run again. If it still does not work, try to  enlarge the patience in the config.json and change the random seed.

**Comment by OfirArviv** [comment]
Thank you for your reply.
Using the exact same packages versions did in fact improved the results! Now I get on the French data an average F1 score of 0.7654 over 5 different runs. Which is a lot closer you your results.
Thanks again.
